examples/mcp/src/elicitation-multi-step/client.ts

```diff
@@ -0,0 +1,173 @@
+import { experimental_createMCPClient, ElicitRequestSchema } from '@ai-sdk/mcp';
```

Comment:
I think for docs and examples, we usually use aliases to remove the `experimental_` from the code
```suggestion
import { experimental_createMCPClient as createMCPClient, ElicitRequestSchema } from '@ai-sdk/mcp';
```